docs/InstallGuide.md

```diff
@@ -39,17 +34,10 @@ Install epel-release.
     # rpm -Uvh http://dlc.wakame.axsh.jp.s3-website-us-east-1.amazonaws.com/epel-release
 
 
-OpenVNet installation
-------------------------
-
-Install OpenVNet.
+Install OpenVNet packages.
 
     # yum install -y openvnet
 
-Initial Configuration
----------------------
-
-### Setup vnmgr
 
 Modify the following section in /etc/openvnet/vnmgr.conf on the vnmgr node.
```

Comment:
Use a \`\` around the file name. Will make it looks like this which makes it easier to distinguish from the other text: `/etc/openvnet/vnmgr.conf`